### PR TITLE
Uskis Arable Land, Spring Light Woods & miscellaneous additions and fixes (Complete)

### PR DESCRIPTION
**Pull request by Sunaries**
Work in progress. Do not merge yet.

@imcgames 

Q1. You translated 패릿 as 'Pharet'. Is it 'Pharet' or 'Ferret'? An alternative option could be: "Pharret"
I need to know this so I can fix the inconsistency. I currently edit it as Ferret.

Q2. 

● '벌들이 속삭이던 분이 당신이군요. 저는 인연의 여신 아우스테야입니다.' 

● '어쩌면 봉인을 지키고 있는 것이 나을지도 모르겠습니다만. 사라진 자매들을 찾는 것이 이 모든 일의 해결책이라 믿고 있습니다.'

↑ But Laima is already the Goddess of Fate. Austeja is the Goddess of Bees, if you want to stick with the original Lithuanian mythology. It's currently translated as 'goddess of fate' and 'in the name of all fate'. Leave it like that?

Q3. 'Darkness/Dark' is a property in TOS. It is also  a status effect, but I've heard of several players that this confuses them. Please see the following example.

**Example**: 

**KR** 스킬 [에피지]에 피격되는 적이 피격될 때마다 특성 레벨당 5% 확률로 5초 동안 [암흑] 상태에 걸립니다.
**ENG** Enemies hit by [Effigy] can be affected by [Darkness] with a 5% chance for 5 seconds per attribute level.

The attribute name is: 에피지: 블라인드 (Effigy: Blind)- It says blind as in it blinds the enemy, making them see darkness. Should we change the [Darkness] status effect to [Blind]?

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -15470,50 +15470,50 @@ ETC_20151001_015472	Survive for 5 minutes!{nl}The mission fails if you're wiped
...
+ETC_20151001_015505	Defeat Clymen!
+ETC_20151001_015506	Clymen has been defeated.{nl}Move on to the next location!
+ETC_20151001_015507	Protect the Paladin Barrier from the Pantos!
+ETC_20151001_015508	Romuva Barrier
+ETC_20151001_015509	Gloomy Panto Assassin
+ETC_20151001_015510	Protect the Romuva Barrier from the Pantos.{nl}Move on to the next location!
+ETC_20151001_015511	Defeat the Gloomy Green Apparition!
+ETC_20151001_015512	The Green Woodspirit has been defeated!
+ETC_20151001_015513	The mission has been completed.{nl}You will return to your original area after 60 seconds.
+ETC_20151001_015514	There are no remaining players left.{nl}You will reuturn to your original area.
```
Comment (WatchGintama) [inline ETC.tsv]: "reuturn"

File: ITEM.tsv [hunk 2]
```diff
@@ -7080,196 +7080,196 @@ ITEM_20151001_007079	Transparent Shell
...
+ITEM_20151016_007194	Increases your AoE Attack Ratio by 3 for 20 seconds. Right-click to use. Cooldown: 60 seconds.
+ITEM_20151016_007195	AoE Attack Ratio Increase Potion Lv2
+ITEM_20151016_007196	Increases your AoE Attack Ratio by 3 for 25 seconds. Right-click to use. Cooldown: 60 seconds.
+ITEM_20151016_007197	AoE Attack Ratio Increase Potion Lv3
+ITEM_20151016_007198	Increases your AoE Attack Ratio by 3 for 30 seconds. Right-click to use. Cooldown: 60 seconds.
+ITEM_20151016_007199	AoE Attack Ratio Increase Potion Lv4
+ITEM_20151016_007200	Increases your AoE Attack Ratio by 3 for 35 seconds. Right-click to use. Cooldown: 60 seconds.
+ITEM_20151016_007201	AoE Attack Ratio Increase Potion Lv5
+ITEM_20151016_007202	Increases your AoE Attack Ratio by 3 for 40 seconds. Right-click to use. Cooldown: 60 seconds.
+ITEM_20151016_007203	When a demon loses its power, it will also its hair, but it may remain in rare circumstances. Obtained from Elma.
```
Comment (WatchGintama) [inline ITEM.tsv]: "it will also its hair" ?

**Comment by WatchGintama** [comment]
Looks good otherwise

**Comment by Sunaries** [comment]
@imcgames Good to merge.

**Comment by imcgames** [comment]
@Sunaries 

Sorry for the late reply, due to too busy now to prepare the upcoming 2nd iCBT.
1. Yes, Please use "Ferret", Actually, It looks really "Ferret Animals".
2. Yes, Please leave it like that, i think the original Korean text is wrong. It should be fixed in the kCBT firstly.
3. 어둠, 암흑 is Dark, Darkness. 블라인드 is Blind. hmm..I think current names looks not bad to show the meaning. But if you want to change it like your idea, then that's okay. xD

**Pull request merged**